third-party-chains/osmosis-types/gamm/pool-models/stableswap/msgs.go

```diff
@@ -4,8 +4,7 @@ import (
 	sdkioerrors "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-
-	"github.com/ingenuity-build/quicksilver/osmosis-types/gamm"
+	gamm2 "github.com/ingenuity-build/quicksilver/third-party-chains/osmosis-types/gamm"
```

Comment:
why `gamm` -> `gamm2`? Most of the changes in the this file and a bunch of the others are unneccessary, only the import needed to change.